ui/src/message_popup/mobile_prototype/texting_interface.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+
+import Divider from 'material-ui/Divider';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+
+import {StudentMessage, UserMessage, InfoMessage} from './message_components.jsx';
+
+export const TextBody = React.createClass({
+  propTypes: {
```

Comment:
Please add in a `displayName` property for all React components so they look nice in debugging messages.